examples/java/spring-boot/build.gradle

```
@@ -28,5 +28,5 @@ processTestResources.finalizedBy copySchemas
 jsonSchema2Pojo {
   source = files(fileTree(dir: '../../../schemas/v1alpha', include: ['*.json']))
   targetDirectory = file("${project.buildDir}/generated-sources/js2p")
-  targetPackage = "models.metadata"
+  targetPackage = "uk.gov.api.models"
```

I know I said let's change it to this on Slack, but actually I think keeping the `.metadata` would be nice
```suggestion
  targetPackage = "uk.gov.api.models.metadata"
```